test/library/packages/Sort/correctness/shellSortExtremeInds.future

```diff
@@ -0,0 +1,6 @@
+bug: shellSort doesn't work when the high bound is too high
+
```

Comment:
Could you please make an issue for this and put the issue number here? It might be a reasonable bug for e.g. GSoC applicants to look at. Thanks.

Reply:
Yeah, I was waiting to make sure you were OK with proceeding without that bug fix or to see if you saw an obvious problem in my first attempt (not that I'm necessarily eager to work on it more).